Home/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>E-com whitepaper</title>
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="mobileNav d-md-none">
+        <i class="fa fa-bars pt-4 fa-3x" id="hamburger" aria-hidden="true"></i>
+        <img id="closeX" class="d-none" src="../img/close.png" alt="" />
+      </nav>
+      <nav class="desktopNav mt-3">
+        <div
+          class="destopNavItems d-flex align-items-center justify-content-between"
+        >
+          <div class="logo ms-5 d-none d-md-block">
+            <a class="logo" href="./index.html"
+              ><img src="../img/logo.jpeg" alt=""
+            /></a>
+          </div>
+          <ul
+            id="desktopList"
+            class="items list-unstyled d-flex gap-4 align-items-center d-none d-md-flex flex-column flex-md-row"
+          >
+            <li>
+              <a class="text-decoration-none text-dark" href="#program"
+                >Program</a
+              >
+            </li>
+            <li>
+              <a class="text-decoration-none text-dark" href="#speakers"
+                >Speakers</a
+              >
+            </li>
+            <li>
+              <a class="text-decoration-none text-dark" href="#sponsors"
+                >Sponsors</a
+              >
+            </li>
+            <li class="ms-5 me-5 border border-danger p-2 bg-danger">
+              <a
+                class="text-decoration-none text-white"
+                href="../About/about.html"
+                >About</a
+              >
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <h3>"Sharing with the world"</h3>
+      <h1 class="col-md-7">THE PERFECT BLUEPRINT TO E-COMMERCE 2022</h1>
+    </header>
+
+    <main class="col-md-12">
+      <div class="hero">
+        <p class="col-md-5 mx-md-5">
+          The long awaited joyful moment beliving in the value of openess and
+          sharing, creating a positive change with different people from accros
+          the world is taking place in August, Abuja Nigeria
+        </p>
+        <div class="date mx-md-5">
+          <h2>2022.08.11(THU) ~ 12(FRI)</h2>
+          <p>@ Ojayent e-comm hub</p>
+        </div>
+      </div>
+
+      <section id="program" class="mainProgram text-white">
```

Comment:
- [ ] I think in your desktop version the main program div in not has the same width and height so kindly make them all the same width and height to match the design as much as possible

- [ ] Kindly Fix the button to be the same in the design so here it doesn't have the background color it is just has  a text  decoration as you can see so please fix it to be the same in the desktop version





- Your design
![image](https://user-images.githubusercontent.com/58553711/181146424-943140f8-1fc8-4a98-8c7b-7771b72457b4.png)


- Original design
![image](https://user-images.githubusercontent.com/58553711/181146261-8d177b46-8f90-4bcf-8ab8-d47baef3ed1d.png)

Reply:
items reposiotioned, icons have been changed